databricks_cli/pipelines/cli.py

```diff
@@ -208,17 +213,45 @@ def reset_cli(api_client, pipeline_id):
 @provide_api_client
 def run_cli(api_client, pipeline_id):
     """
+    (Deprecated) "run" command is deprecated, please use "start" command instead.
+
     Starts the execution of a delta pipelines run by starting the cluster and processing data.
 
     Usage:
 
     databricks pipelines run --pipeline-id 1234
     """
+    click.echo("DeprecationWarning: \"run\" command is deprecated," +
+               " please use \"start\" command instead")
     _validate_pipeline_id(pipeline_id)
-    PipelinesApi(api_client).run(pipeline_id)
+    PipelinesApi(api_client).start(pipeline_id, full_refresh=False)
     click.echo("Run triggered for pipeline {}".format(pipeline_id))
 
 
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Starts the execution of a delta pipeline run')
+@click.option('--pipeline-id', default=None, type=PipelineIdClickType(),
+              help=PipelineIdClickType.help)
+@click.option('--full-refresh', default=False, type=bool,
+              help='If true, truncates tables and creates new checkpoint' +
+                   ' folders so data is reprocessed from scratch')
+@debug_option
+@profile_option
+@pipelines_exception_eater
+@provide_api_client
+def start_cli(api_client, pipeline_id, full_refresh):
+    """
+    Starts the execution of a delta pipelines run by starting the cluster and processing data.
```

Comment:
Ultimately I think this is a decision for engineering and product, but I'll just note that we seem to use different terminology here then is used elsewhere, for example the API uses the "update" request to trigger a pipeline run. "Update" is also consistent with terminology used elsewhere in our DLT documentation.

Also, I think "delta pipelines" is a reference to the old name? Should this be "a Delta Live Tables pipeline run", or just "a pipeline run"?

Reply:
I like `update`. Should have thought of this earlier.

Reply:
changed the command to "update".

Also, updated "delta pipelines run" to "a pipeline run" in most places in the file.

Reply:
@jaseidman we had a long conversation w.r.t to whether naming the command to start an update as "start", "update", "start update". For now, we decided to go with start as it makes sense in both triggered update and continuous update case and it keeps the symmetry between start/stop. 

I will revert the naming from update => start